src/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,95 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import styles from './EventCard.module.css';
+import { Button } from 'react-bootstrap';
+import React from "react";
+import Image from 'next/image'
+
+function EventCard( {title, image, author, authorPic, maxCap, currCap, location, time, description, joined, owner} ) {
+    EventCard.propTypes = {
+        title: String,
+        image: String,
+        author: String,
+        authorPic: String,
+        maxCap: Number,
+        currCap: Number,
+        location: String,
+        time: String,
+        description: String,
+        joined: Boolean,
+        owner: Boolean,
+    }
+    let buttText;
+    if (joined) {
+        buttText = "Happening Soon!";
+    } else {
+        buttText = "Join!"
```

Comment:
as funny as it is name buttText to buttonText

Reply:
Fixed